README.md

```diff
@@ -5,13 +5,13 @@ gluster Cookbook
 
 This cookbook is used to install and configure Gluster on both servers and clients. This cookbook makes several assumptions when configuring Gluster servers:
 
-1. If using the cookbook to format disks, each disk will contain a single partition dedicated for Gluster
```

Comment:
README should also mention `v5.0.0` (what this rewrite will become) is not backwards compatible with `v4.0.2` due to the addition of LVM